docs/gen-1/red-blue/main-glitchless/race-route/README.md

```diff
@@ -536,20 +539,20 @@ NOTE: From this point on you have 2 revives which means deaths aren't as scary,
 * Growlithe: X ACC, EQ
   - If Blaine use Super Potion turn 1, Horn Drill Growlithe instead
 * HD all remaining pokes
-- Digout and Bike to Sabrina's Gym
+- Dig out and Bike to Sabrina's Gym
 
-##Saffaron City
+## Saffron City
 - Teleporter Puzzle: Top Left, Bottom Left, Bottom Left
 
 **Sabrina:**
 - EQ x4
-- Walk back to the Teleporter and dig out and fly to
+- Walk back to the Teleporter and Dig out and Fly to
 
 ## Viridian City
 **Cooltrainer:**
 - EQ
 
-SAVE
+SAVE  
```

Comment:
is this change needed?

Reply:
Same as above (current doc misses a new line)